starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,33 @@
+body {
+    margin: 0px;
+    font-family: open sans;
+}
+
+h1 {
+    text-align: center;
+    padding-top: 500px;
+    color: white;
+    font-size: 100px;
+}
+
+.page-container {
+    margin: 200px 200px 100px 200px;
+    background-color: white;
+    position: relative;
+}
+
+.recipes-cooking-info {
+    text-align: center;
+}
+
+.background-pie {
+    background-image: url(/Users/pedroneto/Desktop/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg);
```

Comment:
Use relative paths instead of absolute ones, the same way you have done it with 'cooking-info' and 'reciping-info'. Now the background only displays in your computer.